components/application-broker/internal/broker/provision_test.go

```diff
@@ -655,6 +665,9 @@ func TestDoProvision(t *testing.T) {
 				bt.NewAppNamespace(string(appNs), false),
 				bt.NewAppSubscription(string(appNs), string(appName)),
 			},
+			expectCreates: []runtime.Object{
+				bt.NewIstioPolicy(string(appNs), string(appNs)+policyNameSuffix),
```

Comment:
It turns out `NewIstioPolicy` is only used for testing. I think a better name(e.g. `createTestPolicy`) should be used to clear things out. What do you think?

Reply:
@sayanh I'd say it's already in the testing package, and the second thing, I'm following the convention of naming helpers in the same file.